src/msa.js

```diff
@@ -61,7 +62,10 @@ const MSA = boneView.extend({
     this.g.vis = new Visibility(data.vis, {model: this.seqs});
     this.g.visorder = new VisOrdering(data.visorder);
     this.g.zoomer = new Zoomer(data.zoomer,{g:this.g, model: this.seqs});
-
+    
+    // store config options for plugins
+    this.g.conservationConfig = data.conserv;
```

Comment:
just an FYI (no need to change): the other plugins use backbone models, because one can listen to change events there, but as I plan to use Object.observe (or similar) in the future - this shouldn't be a problem ;-)